.github/workflows/post-commit.yml

```diff
@@ -19,9 +19,19 @@ jobs:
   build:
     name: compile
     runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        java: [ 8, 11 ]
```

Comment:
Out of curiosity: is this the setup to have post commits runs build by java 11? If so, is there a way to trigger the post commit in this PR so this change can be verified?

Reply:
> is this the setup to have post commits runs build by java 11?

Correct.

> If so, is there a way to trigger the post commit in this PR so this change can be verified?

It was [already triggered](https://github.com/adoroszlai/incubator-ratis/runs/1385567774) in my own fork.

Reply:
Thank you for the clarification!